sample/src/androidTest/java/com/schibsted/spain/barista/sample/FlakyRuleTest.java

```diff
@@ -51,7 +51,9 @@ public void someFlakyTest() throws Exception {
     }
   }
 
+  // WARNING: this test is flaky. Avoid running it at the C.I.
   @Test
+  @Ignore
```

Comment:
Don't you want to use your new annotation here? From the comment it looks like you do.